app/main.py

```diff
@@ -2,0 +1,14 @@
+
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+    if command_parts[0] != "cp":
+        print("Invalid command format.")
+        return
+    first_txt, second_txt = command_parts[1], command_parts[2]
```

Comment:
Code Style: The variable names `first_txt` and `second_txt` are not descriptive enough. Consider renaming them to something more meaningful, such as `source_file` and `destination_file`.